celements-scheduler/src/main/java/com/celements/scheduler/SchedulerPlugin.java

```diff
@@ -331,15 +222,12 @@ public boolean scheduleJob(BaseObject object, XWikiContext context)
       JobDetail job = new JobDetail(xjob, Scheduler.DEFAULT_GROUP,
           Class.forName(object.getStringValue("jobClass")));
       Trigger trigger = new CronTrigger(xjob, Scheduler.DEFAULT_GROUP, xjob,
-          Scheduler.DEFAULT_GROUP,
-          object.getStringValue("cron"));
-      // Let's prepare an execution context...
-      XWikiContext stubContext = prepareJobStubContext(object, context);
-      data.put("context", stubContext);
-      data.put("xcontext", stubContext);
-      data.put("xwiki", new com.xpn.xwiki.api.XWiki(context.getWiki(), stubContext));
-      data.put("xjob", object);
-      data.put("services", Utils.getComponent(ScriptServiceManager.class));
+          Scheduler.DEFAULT_GROUP, object.getStringValue("cron"));
+      data.put("jobDoc", modelAccess.get().getDocument(object.getDocumentReference()));
```

Comment:
only set the jobDocRef in the data map and fetch it in the job-thread instead of cloning.